lzy/worker/docker/UserDefault.Base.Dockerfile

```diff
@@ -4,9 +4,15 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 ### deps
 RUN apt-get update && \
-    apt-get install -y wget sox libsndfile1 ffmpeg libgomp1 && \
+    apt-get install -y wget locales sox libsndfile1 ffmpeg libgomp1 && \
```

Comment:
This is not about your PR, but we are missing apt cache cleaning here

Reply:
But cache is actually cleaned in the next line.
`rm -rf /var/lib/apt/lists/*`

Reply:
My bad